### DISCUSSION TITLE
CORS Configuration

### DISCUSSION THREAD

**Discussion by jon9595** [stac-utils/stac-fastapi#664] [General]
Hello! I'm new to using stac-fastapi, and I am implementing it using the stac-fastapi-sqlalchemy backend. I see in the `app.py` file of `stac-fastapi` that the CORSMiddleware is used in a lambda, but I am a bit lost on how to configure it for specific URLs. My front end is having CORS issues when connecting to it currently.

**Comment by pedro-cf** [comment]
Greetings @jon9595 
The approach I've used:

```python
from starlette.middleware import Middleware

api = StacApi(...)

api.add_middleware(Middleware(
        CORSMiddleware,
        allow_origins=["*"],
        allow_credentials=True,
        allow_methods=["*"],
        allow_headers=["*"]
    )
)
```
